Gems/Atom/RPI/Code/Include/Atom/RPI.Public/ViewportContextBus.h

```diff
@@ -42,6 +42,8 @@ namespace AZ
                 RHI::Device* device = nullptr;
                 //! The native window to create a swap chain for, must be valid.
                 AzFramework::NativeWindowHandle windowHandle = {};
+                // xcb_connection_t on platforms using X11/xcb
+                void* nativeConnection = nullptr;
```

Comment:
Seems like the `WSISurface::Descriptor` struct can be used as a member here instea of having a separate Window and NativeConnection handler